### PR TITLE
Add Pluto package files

### PR DESCRIPTION
**Pull request by jziegel**
**Favro**
https://favro.com/organization/c9668dc39f1bd49fed254201/64b92cf78f476352eee739e7?card=Cre-3018

### CODE REVIEW

File: packages/pluto/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "jest": "^24.9.0",
+    "prettier": "^1.19.1",
+    "selenium-webdriver": "4.0.0-alpha.5",
+    "ts-jest": "^24.2.0",
+    "ts-node": "8.6.2",
+    "typescript": "^3.7.2",
+    "webpack": "^4.41.2"
+  },
+  "dependencies": {
+    "@types/selenium-webdriver": "4.0.6",
```
Comment (mnmtanish) [inline packages/pluto/package.json]: Typings should go under `devDependencies`.

File: packages/pluto/readme.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Pluto
+
+### An end to end testing framework.
+
+## Setting up for local development
+
+### Clone this repository
```
Comment (mnmtanish) [inline packages/pluto/readme.md]: The readme is not user friendly. Assume the person who is going to use it is not working with us. They just want to install this tool and run some tests. Some example tests will help.

File: packages/pluto/readme.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+```shell
+git clone git@github.com/creately/modules-js.git
+cd modules-js/packages/pluto
+npm install
```
Comment (mnmtanish) [inline packages/pluto/readme.md]: It should be possible to install pluto with.

```
npm install -g @creately/pluto
```

File: packages/pluto/src/__module.d.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// module declarations
+declare module 'md5';
```
Comment (mnmtanish) [inline packages/pluto/src/__module.d.ts]: Why do we need both? and I couldn't find them on package.json.

File: packages/pluto/package.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  },
+  "preferGlobal": true,
+  "author": "Creately",
+  "license": "MIT",
+  "devDependencies": {
+    "@types/jest": "^24.0.23",
+    "@types/lodash": "4.14.149",
+    "jest": "^24.9.0",
+    "prettier": "^1.19.1",
+    "selenium-webdriver": "4.0.0-alpha.5",
```
[the remark this answers is not in the document]
Comment (jziegel) [reply packages/pluto/package.json]: Noted, done.

File: packages/pluto/package.json (hunk 1 above)
Comment (jziegel) [reply packages/pluto/package.json]: Noted, done.

File: packages/pluto/src/__module.d.ts (hunk 4 above)
Comment (jziegel) [reply packages/pluto/src/__module.d.ts]: We don't, this seems to be left over from the package split.

File: packages/pluto/readme.md (hunk 3 above)
Comment (jziegel) [reply packages/pluto/readme.md]: Noted, thanks.

File: packages/pluto/readme.md (hunk 2 above)
Comment (jziegel) [reply packages/pluto/readme.md]: I'll update it, thanks.

**Pull request merged**